TTTSwiftCommandLine/Game.swift

```diff
@@ -1,9 +1,10 @@
 public class Game {
     
-    public var board: Board!
-    
+    let board: Board!
     let clock: Clock!
     let rules: Rules!
+    let boardPrinter: ConsoleBoard!
```

Comment:
I would definitely avoid using the `!` in your POSO (Plain Old Swift Objects). Doing so removes the safety of having optionals in the first place.

If you have to have an optional in a POSO, I would definitely favor using a `?`, although the ideal would be to limit your use of optionals to the iOS specific code. e.g. In your `UIView` and `UIViewController` classes.